src/mito2/src/worker/handle_flush.rs

```diff
@@ -184,12 +184,13 @@ impl<S: LogStore> RegionWorkerLoop<S> {
         );
         region.update_flush_millis();
 
+        let timer = std::mem::take(&mut request.timer);
+        let elapsed = timer.map(|x| x.stop_and_discard()).unwrap_or_default();
```

Comment:
issue: This behavior is incorrect: It stops the timer earlier. Since the timer doesn't have an `elapsed()` method, we can remove the `elapsed` time from the log. We might wrap a new timer in the future to support more features.

Reply:
Ok